com.ibm.wala.cast.js/src/main/java/com/ibm/wala/cast/js/util/FieldBasedCGUtil.java

```diff
@@ -205,9 +228,46 @@ public CallGraphResult buildCG(
     com.ibm.wala.cast.util.Util.checkForFrontEndErrors(cha);
     Iterable<Entrypoint> roots = JSCallGraphUtil.makeScriptRoots(cha);
     IAnalysisCacheView cache = new AnalysisCacheImpl(AstIRFactory.makeDefaultFactory());
+    final FieldBasedCallGraphBuilder builder;
+    if (builderType.toString() == "OPTIMISTIC_WORKLIST") {
+      builder =
+          new WorklistBasedOptimisticCallgraphBuilder(
+              cha,
+              JSCallGraphUtil.makeOptions(scope, cha, roots),
+              cache,
+              supportFullPointerAnalysis,
+              -1);
+    } else {
```

Comment:
I don't think we need the `if` case, due to the change you made on line 89 of `FieldBasedCGUtil`